styles.css

```diff
@@ -1,5 +1,7 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
 
+/* Mobiles & Tablets */
+
```

Comment:
- [Optional] Dear @mdg0410, Also your design is great I see your design is not responsible for screen size between `375px` to `767px`, Please consider removing the fixed width for the mobile version so it can be responsive to all screens in between. remember people are going to visit your portfolio on a different screen. 

### Your design on a 600px screen
![Screenshot 2023-01-27 084824](https://user-images.githubusercontent.com/69077061/215018429-3ab49e46-e940-461c-9d40-acf4a3f3c527.png)

Reply:
Hi, @Tadesse-Alemayehu 

I would like to thank you for your observation, it motivated me to correct and improve my project.

> Correction

Eliminate fixed sizes and let display flex do its thing

|Mobile Version|Mobile Version|
|:---------------:|:----------------:|
|375px|760px |
|![imagen](https://user-images.githubusercontent.com/90431831/215189956-f6903318-9209-410f-885e-6ec859c653fa.png)|![imagen](https://user-images.githubusercontent.com/90431831/215190127-3c9efb76-60ef-4308-b63d-e536c0eb623b.png)|

Reply:
### good job @mdg0410 🙌